charts/sam/templates/deployment.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: sam-deployment
+  {{- template "sam.labels" }}
+spec:
+  # TODO consider changing the replicas number or always overlaying it for different environments.
```

Comment:
doesn't have to be this PR, but this is an example of something we would add a chart variable for, right?

Reply:
Yep, I'll add it. Mostly I just copy/pasted stuff and got rid of the kustomize-specific bits

Reply:
remove the comment too.